lib/registrant.rb

```diff
@@ -0,0 +1,42 @@
+class Registrant
+  attr_reader :name,
+              :age,         # getter method
+              :license_data
+
+  def initialize(name, age, permit = false)
+    @name = name
+    @age = age
+    @permit = permit
+    @license_data = {:written => false, :license => false, :renewed => false}
+  end
+
+  def permit?
```

Comment:
Great job with this one and great job NOT putting `:permit` in the attr_reader which many people did. We don't want it in the attr_reader since we have it here.